docs/tutorials/geomesa-blobstore.rst

```diff
@@ -0,0 +1,150 @@
+GeoMesa BlobStore
+=================
+
+This tutorial will show you:
+
+1. What the BlobStore is
+2. How to extend the BlobStore with FileHandlers
+3. How to deploy the GeoMesa BlobStore Servlet
+4. How to add FileHandlers to the servlet deploy
+5. How to use the RESTful API
+
+Prerequisites
+-------------
+
+.. warning::
+
+    For Accumulo deployment, you will need access to an Accumulo |accumulo_version| instance.
```

Comment:
Yeah, if we want to have more explicit min/max Accumulo versions, we'll have to change the other tutorials too.